README.md

```diff
@@ -62,16 +62,23 @@ android {
 ```
 
 ## Starting the PhotoEditor activity
-Simply add this line to the place from where you want to start the activity-
 
+In the activity from where you want to start the photo editor add this line outside all methods in your activity class-
```

Comment:
`Add this global static constant to your activity class with your preferred request code:`